deploy/templates/nstemplatetiers/base1ns/cluster.yaml

```diff
@@ -3,25 +3,6 @@ kind: Template
 metadata:
   name: base1ns-cluster-resources
 objects:
-- apiVersion: quota.openshift.io/v1
-  kind: ClusterResourceQuota
-  metadata:
-    name: for-${USERNAME}-compute
-  spec:
-    quota:
-      hard:
-        limits.cpu: 20000m
-        limits.memory: ${MEMORY_LIMIT}
```

Comment:
I think you removed the `MEMORY_LIMIT` below? You'd have to either add it back or fill in the value here instead.

Reply:
This clusterresourcequota was deleted, hence MEMORY_LIMIT and MEMORY_REQUEST were deleted below.

Reply:
Not sure why I thought this was an add and not a remove 🤦 My bad...